src/main/java/com/google/gcp_variant_transforms/common/Constants.java

```diff
@@ -24,4 +24,12 @@
     public static final String CALLS_GENOTYPE = "genotype";
     public static final String CALLS_PHASESET = "phaseset";
   }
+
+  /**
+   * Commonly used VCF Format Header Line IDs
+   */
+  public static class VCFFormatIDNames {
```

Comment:
Those field names can be found in [VCFConstants](https://github.com/samtools/htsjdk/blob/d5ac8634fa7a74c0275842ff3826c819259d5d12/src/main/java/htsjdk/variant/vcf/VCFConstants.java)

Reply:
Modified the code to use VCFConstants instead. Thanks!